app/src/db/models/document.py

```diff
@@ -21,6 +21,12 @@ class Document(Base, IdMixin, TimestampMixin):
         "Chunk", back_populates="document", cascade="all, delete"
     )
 
+    # Domain-specific columns follow
+    # benefit program
+    program: Mapped[str | None]
+    # geographical region of the benefit program
+    region: Mapped[str | None]
```

Comment:
Not something I feel super strongly about, but what do you think about making these non-optional?